test_runner/random_test_runner/Readme.md

```diff
@@ -0,0 +1,213 @@
+# Random test runner
+
+Random test runner allows running randomly generated scenarios to test autoware autonomy implementation.
+
+| NOTE: Currently only `autoware.auto` (`awf/auto`) is supported. `AutowareArchitectureProposal` (`tier4/proposal`) will be supported in the future. |
+|----------------------------------------------------------------------------------------------------------------------------------------------------|
+
+## How to build
+
+### Autoware.auto
+
+It is assumed that you are running AutowareAuto in ADE environment with default paths. See [Autoware.Auto documentation](https://autowarefoundation.gitlab.io/autoware.auto/AutowareAuto/installation-ade.html) for instructions how to correctly set up Autoware.Auto and ADE.
+
+Inside Autoware.auto directory run/enter ade:
+```shell
+ade --rc .aderc-amd64-foxy-lgsvl start --update --enter
+```
+
+enter autoware directory, install required dependencies and build
+
+```shell
+cd <autoware_auto_directory_name>
+vcs import < autoware.auto.foxy.repos
+./tools/simulation/get_scenario_simulator_v2.sh
+colcon build --packages-up-to random_test_runner scenario_simulator_launch kashiwanoha_map --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Comment:
I need to build `openscenario_visualization` and `behavior_tree_plugin` separately, probably there are some missing dependencies

Reply:
Does it build as described now?

Reply:
Adding `behavior_tree_plugin` dependency in `traffic_simulator` creates circular dependency (`behavior_tree_plugin` already depends on `traffic_simulator`). In this case I simply added `behavior_tree_plugin` to `random_test_runner`'s `package.xml` runtime dependency.